src/tests/ftest/util/test_utils.py

```diff
@@ -27,12 +27,16 @@
 
 from avocado import fail_on
 from avocado.utils import process
-from command_utils import BasicParameter, ObjectWithParameters
+from command_utils import BasicParameter, ObjectWithParameters, CommandFailure
 from pydaos.raw import (DaosApiError, DaosServer, DaosContainer, DaosPool,
-                        c_uuid_to_str)
+                        c_uuid_to_str, daos_cref)
 from general_utils import check_pool_files, get_random_string, DaosTestError
 from env_modules import load_mpi
-
+import ctypes
+import getpass
+import grp
+from dmg_utils import (get_pool_uuid_from_stdout,
```

Comment:
(pylint-no-name-in-module) No name 'get_service_replicas_from_stdout' in module 'dmg_utils'